config/postgres/default_postgresql.conf

```diff
@@ -5,33 +5,32 @@ listen_addresses = 'localhost'
 port = 5432
 compute_query_id = on
 
-# Extensions
-shared_preload_libraries = 'auto_explain,pg_stat_statements'
-auto_explain.log_min_duration = '-1'
-auto_explain.log_analyze = 'false'
-auto_explain.log_format = 'json'
-auto_explain.log_verbose = 'true'
-pg_stat_statements.max = 10000
-pg_stat_statements.track = all
-pg_stat_statements.save = off
-
-# PGTune suggested these parameters for group's default Intel NUCs
+# PGTune suggested these parameters for (dev4)
+# https://pgtune.leopard.in.ua/
+#   - DB Version: 14
+#   - OS Type: Linux
+#   - DB Type: Mixed Type of Application
+#   - Total Memory: 128 GB
+#   - Number of CPUs: 40
+#   - Number of Connections: 200
+#   - Data Storage: SSD
 max_connections = 200
-shared_buffers = 8GB
-effective_cache_size = 24GB
+shared_buffers = 32GB
```

Comment:
Switched the default to a more reasonable laptop assumption. Decided to add a separate `dev4_postgresql.conf` file since `noisepage_init` supports starting with a specific configuration.